src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/BaseCCJPdfTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional.citizen;
+
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment.AmountContent;
+import uk.gov.hmcts.cmc.claimstore.tests.functional.BasePdfTest;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.Interest;
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.amount.AmountBreakDown;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+
+public abstract class BaseCCJPdfTest extends BasePdfTest {
+
+    protected void assertHeaderDetails(Claim caseGiven, String pdfAsText, String expectedTitle) {
+        assertThat(pdfAsText).contains(expectedTitle);
+        assertThat(pdfAsText).contains("In the County Court Business Centre");
+        assertThat(pdfAsText).contains("Online Civil Money Claims");
+        assertThat(pdfAsText).contains("Claim number: " + caseGiven.getReferenceNumber());
+        assertThat(pdfAsText).contains("Requested: " + formatDate(caseGiven.getCountyCourtJudgmentRequestedAt()));
+    }
+
+    protected void assertPartyAndContactDetails(ClaimData claimData, String pdfAsText) {
+        assertThat(pdfAsText).contains("Claimant details");
+        assertThat(pdfAsText).contains("Name: " + claimData.getClaimant().getName());
+        assertThat(pdfAsText).contains("Address: "
+            + getFullAddressString(claimData.getClaimant().getAddress()));
+        claimData.getClaimant().getMobilePhone()
+            .ifPresent(value -> assertThat(pdfAsText).contains("Telephone: " + value));
```

Comment:
assert also if the getMObilePhone() is not present, the pdftext doenst have the unexpected value as well.